README.md

```diff
@@ -526,3 +501,45 @@ see: `Utils.clean` in
 [/lib/utils.ex#L17](https://github.com/dwyl/quotes/blob/cff02564691b512e66a2d39701fcd0d6b19d98f3/lib/utils.ex#L17)
 
 -->
+
+### Disclaimer for contributions
+
+If you are thinking of _contributing_ to this repository (hooray! 🎉),
+please make sure you
+[read the `Contributing` section](https://github.com/dwyl/quotes#contributing)
+of this repo.
+
+There are many websites that already contain quotes
+and this repository is
+intended as a _subset_ of uniquely _inspiring_ quotes.
+See [`Avoid Quotes That Don't Inspire Action`](https://github.com/dwyl/quotes#avoid-quotes-that-dont-inspire-action).
+
+As such, **_we remind you that we will only be able to accept PRs that conform
```

Comment:
YES. This section is SO necessary ❤️  People have just been using this repo as a green squares machine during hacktoberfest 🙄 
Thank you for adding it.